docs/docs/05-reusable-components.md

```diff
@@ -232,3 +232,25 @@ Methods follow the same semantics as regular ES6 classes, meaning that they don'
 ### No Mixins
 
 Unfortunately ES6 launched without any mixin support. Therefore, there is no support for mixins when you use React with ES6 classes. Instead, we're working on making it easier to support such use cases without resorting to mixins.
+
+
+
+## Stateless Functions
+
+You may also define your React classes as a plain JavaScript function. For example using the stateless function syntax:
+
+```javascript
+function HelloMessage(props) {
```

Comment:
Ok, sounds good.  I think it's worth having an example that uses the ES5 syntax, since it's more familiar to people and ES6 isn't widely supported, but I just added an ES6 example to go alongside.